compose/src/main/java/com/bottlerocketstudios/compose/repository/RepositoryBrowserScreen.kt

```diff
@@ -179,3 +171,10 @@ private val browserItems = listOf(
         isFolder = false
     ),
 )
+
+val previewState = RepositoryBrowserScreenState(
```

Comment:
@BottleRocket-Colin nit: Make this `private`

```suggestion
private val previewState = RepositoryBrowserScreenState(
```